tdrs-frontend/src/components/Footer/Footer.jsx

```diff
@@ -5,6 +5,24 @@ import ACFLogo from '../../assets/ACFLogo.svg'
 function Footer() {
   return (
     <footer className="usa-footer usa-footer--slim">
+      <div className="usa-footer__primary-section">
+        <div className="usa-footer__primary-container grid-row">
+          <div className="mobile-lg:grid-col-8">
+            <nav className="usa-footer__nav" aria-label="Footer navigation">
+              <ul className="grid-row grid-gap">
+                <li className="mobile-lg:grid-col-6 desktop:grid-col-auto usa-footer__primary-content">
+                  <a
+                    className="usa-footer__primary-link"
+                    href="https://www.acf.hhs.gov/privacy-policy"
+                  >
+                    Privacy policy
+                  </a>
+                </li>
+              </ul>
+            </nav>
+          </div>
+        </div>
+      </div>
```

Comment:
Closely models the USWDS [slim footer](https://designsystem.digital.gov/components/footer/)